tests/Functional/ConnectionsTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the WouterJEloquentBundle package.
+ *
+ * (c) 2014 Wouter de Jong
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace WouterJ\EloquentBundle\Functional;
+
+use AppBundle\Model\Book;
+use AppBundle\Model\Category;
+use AppBundle\Model\Conn2Book;
+use AppBundle\Model\Isbn;
+use Illuminate\Database\Schema\Blueprint;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use WouterJ\EloquentBundle\Facade\Schema;
+
+/**
+ * @author Wouter J <wouter@wouterj.nl>
+ */
+class ConnectionsTest extends KernelTestCase
+{
+    protected static function getKernelClass()
+    {
+        return 'TestKernel';
+    }
+
+    protected function setUp()
+    {
+        static::bootKernel();
+
+        Schema::create('books', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            $table->integer('isbn_id')->nullable();
+            $table->integer('category_id')->nullable();
+            $table->timestamps();
+        });
+
+        Schema::connection('conn2')->create('books', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            $table->integer('isbn_id')->nullable();
+            $table->integer('category_id')->nullable();
+            $table->timestamps();
+        });
+    }
+
+    public function testMultipleConnections()
+    {
+        $book1 = new Book;
+        $book1->title = 'Hello world!';
+        $book1->save();
+
+        $book2 = new Conn2Book;
+        $book2->title = 'Other world!';
+        $book2->save();
+
+        $this->assertEquals(1, Book::all()->count());
```

Comment:
`assertSame()` is preferable here otherwise you are subject to the errors introduced by weak equality (e.g. stringable objects)

Reply:
In this case, I'm comparing a count with a count. I don't think much type errors can occur here

Reply:
Sure, but I just recommend to use `assertSame()` by default to not have to think of when to use one or the other